contracts/StorageMarket_flat.sol

```diff
@@ -1877,31 +1883,42 @@ contract StorageMarketPlace is Sablier {
         mapping(address => bool) buyers;
     }
 
-    struct BuyerSubscription {
-        address buyer;
-        uint256 amount;
-        uint256 streamId;
-        bool isActive;
-    }
+    // struct BuyerSubscription {
+    //     address buyer;
+    //     uint256 amount;
+    //     uint256 streamId;
+    //     bool isActive;
+    // }
 
-    struct SellerSubscription {
-        address seller;
-        uint256 amount;
-        uint256 streamId;
-        bool isActive;
+    // struct SellerSubscription {
+    //     address seller;
+    //     uint256 amount;
+    //     uint256 streamId;
+    //     bool isActive;
+    // }
+    
+    
+    
+    struct StreamInfo {
+        // extra field cannot keep just an array in struct get this error TypeError: Internal or recursive type is not allowed for public state variables.
+        string text;
```

Comment:
we could replace with a boolean if we dont need it. 
or maybe a uint representing the size of the array